docker/pulsar/pom.xml

```diff
@@ -48,15 +48,23 @@
   </dependencies>
 
   <properties>
-      <UBUNTU_MIRROR>mirror://mirrors.ubuntu.com/mirrors.txt</UBUNTU_MIRROR>
-      <UBUNTU_SECURITY_MIRROR>http://security.ubuntu.com/ubuntu/</UBUNTU_SECURITY_MIRROR>
+    <UBUNTU_MIRROR>mirror://mirrors.ubuntu.com/mirrors.txt</UBUNTU_MIRROR>
+    <UBUNTU_SECURITY_MIRROR>http://security.ubuntu.com/ubuntu/</UBUNTU_SECURITY_MIRROR>
+    <git.commit.id.abbrev>no-git</git.commit.id.abbrev>
   </properties>
 
   <profiles>
     <profile>
       <id>docker</id>
       <build>
         <plugins>
+          <plugin>
+            <groupId>pl.project13.maven</groupId>
+            <artifactId>git-commit-id-plugin</artifactId>
+            <configuration>
+              <skip>false</skip>
+            </configuration>
+          </plugin>
```

Comment:
I think that this should be defined in `docker/pom.xml` so that it gets applied to both `docker/pulsar/pom.xml` and `docker/pulsar-all/pom.xml`.